vpr/src/util/vpr_utils.cpp

```diff
@@ -578,21 +622,35 @@ int find_pb_pin_clb_pin(ClusterBlockId clb, int pb_pin) {
     auto& cluster_ctx = g_vpr_ctx.clustering();
     auto& place_ctx = g_vpr_ctx.placement();
 
+    auto& clb_nlist = cluster_ctx.clb_nlist;
+
+    t_type_ptr equivalent_type = clb_nlist.block_type(clb);
+    t_type_ptr original_type = clb_nlist.block_type(clb, false);
+
+    bool eq_type_selected = original_type == equivalent_type ? false : true;
```

Comment:
Can you explain where there is an "original_type" at all?  Why not always index into the equiv types?

Reply:
I have added additional documentation to this and changed the way the pin is selected. Basically the `original_type` contains the mapping information with the equivalent tiles, that is why it is needed.